src/views/CronTabList/CronTabList.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { CronTabModel } from '@crontab-model/CronTabModel';
+import { CronTabKind } from '@crontab-model/CronTabModel';
+import { modelToGroupVersionKind, modelToRef } from '@crontab-utils/utils';
+import {
+  K8sResourceCommon,
+  ListPageBody,
+  ListPageCreate,
+  ListPageFilter,
+  ListPageHeader,
+  Timestamp,
+  useK8sWatchResource,
+  useListPageFilter,
+  VirtualizedTable,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { useCronTabTranslation } from '@crontab-utils/hooks/useCronTabTranslation';
+import { ResourceLink, RowProps, TableData } from '@openshift-console/dynamic-plugin-sdk';
+import { TableColumn } from '@openshift-console/dynamic-plugin-sdk';
+import { sortable } from '@patternfly/react-table';
+
+import CronTabRowActions from './CronTabRowActions';
+
+
+type CronTabListProps = {
+  namespace: string;
+};
+
+const CronTabList: React.FC<CronTabListProps> = ({ namespace }) => {
+  const [cronTabs, loaded, loadError] = useK8sWatchResource<K8sResourceCommon[]>({
+    isList: true,
+    groupVersionKind: {
+      group: 'stable.example.com',
+      kind: 'CronTab',
+      version: 'v1',
+    },
+    namespaced: true,
+    namespace,
+  });
+  const { t } = useCronTabTranslation();
+  const columns = useCronTabColumns();
+  const [data, filteredData, onFilterChange] = useListPageFilter(cronTabs);
+
+  return (
+    <>
+      <ListPageHeader title={t('CronTab')}>
+        <ListPageCreate groupVersionKind={modelToRef(CronTabModel)}>Create CronTab</ListPageCreate>
+      </ListPageHeader>
+      <ListPageBody>
+        <ListPageFilter data={data} loaded={loaded} onFilterChange={onFilterChange} />
+        <VirtualizedTable<K8sResourceCommon>
+          data={filteredData}
+          unfilteredData={cronTabs}
+          loaded={loaded}
+          loadError={loadError}
+          columns={columns}
+          Row={cronTabListRow}
+        />
+      </ListPageBody>
+    </>
+  );
+};
+
+const cronTabListRow: React.FC<RowProps<CronTabKind>> = ({ obj, activeColumnIDs }) => {
+  return (
+    <>
+      <TableData id="name" activeColumnIDs={activeColumnIDs} className="pf-m-width-15">
```

Comment:
Do we need these width classes? By default the columns should all be the same width?

Reply:
removed